kolibri/core/exams/serializers.py

```diff
@@ -28,7 +28,7 @@ class Meta:
 
 class QuestionSourceSerializer(Serializer):
     exercise_id = HexUUIDField(format="hex")
-    question_id = CharField()
+    question_id = HexUUIDField(format="hex", required=False)
```

Comment:
Is it really possible to have an optional `question_id`?

Reply:
because of the V0:
        JSON array describing exercise nodes this exam draws questions from,
        how many from each, and the node titles at the time of exam creation:
       `[
            {
                "exercise_id": <exercise_pk>,
                "number_of_questions": 6,
                "title": <exercise_title>
            },
            ...
        ]`

Reply:
I see! Could you add a comment above the field explaining that?

Reply:
I think we should actually enforce this - whenever a V0 quiz is loaded into the frontend, it should be converted into a V1 quiz by the frontend code, so if anyone edits a V0 quiz, it will be resaved as a V1 quiz, so validation should only be needed for V1.

Reply:
I'll do that